### DISCUSSION TITLE
Seeking help regarding what exactly alpha and beta refer to when merging models, and the differences between the two？

### DISCUSSION THREAD

**Discussion by IreneDing123** [hako-mikan/sd-webui-supermerger#375] [Q&A]
![20240421122658](https://github.com/hako-mikan/sd-webui-supermerger/assets/134567814/ef22ce15-cdc5-40eb-83b3-b275b16ce4e4)
When merging models, there is a lack of understanding of what the beta value specifically refers to. For instance, if models A and B are adjusting the alpha values, what does the beta value in model C refer to?
![20240421123658](https://github.com/hako-mikan/sd-webui-supermerger/assets/134567814/8f08d088-1fd2-4b10-99e7-51f7125b849e)

**Comment by StanleyPain** [comment]
They refer to the models being merged.

Example. You are merging model (x) and model (y) only "alpha" will be used and it dictates the ratio of model (y) to merge.

You are merging model (x) and model (y) and model (z)  alpha will be the ratio of model y, and beta will be the ratio of model z
